### PR TITLE
Cut release 1.11.7 for Firestore emulator

### PR DESCRIPTION
**Pull request by malcolmdeck**
### Description

This PR updates Firestore Emulator to v1.11.7.

### Scenarios Tested

```bash
$ node lib/bin/firebase.js emulators:start --only firestore
⚠  Could not find config (firebase.json) so using defaults.
i  emulators: Starting emulators: firestore
⚠  firestore: Did not find a Cloud Firestore rules file specified in a firebase.json config file.
⚠  firestore: The emulator will default to allowing all reads and writes. Learn more about this option: https://firebase.google.com/docs/emulator-suite/install_and_configure#security_rules_configuration.
i  firestore: downloading cloud-firestore-emulator-v1.11.7.jar...
Progress: ==================================================================================================================> (100% of 64MB)
i  firestore: Removing outdated emulator files: cloud-firestore-emulator-v1.11.6.jar
i  firestore: Firestore Emulator logging to firestore-debug.log
```

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1 +1,2 @@
 - Fixes an issue where immutable params were not sent during ext:configure.
+- Includes latest features and improvements from production.
```
Comment (yuchenshi) [inline CHANGELOG.md:2]: ...in Firestore Emulator.

File: src/emulator/downloadableEmulators.ts [hunk 2]
```diff
@@ -40,14 +40,14 @@ const DownloadDetails: { [s in DownloadableEmulators]: EmulatorDownloadDetails }
...
+    downloadPath: path.join(CACHE_DIR, "cloud-firestore-emulator-v1.11.6.jar"),
+    version: "1.11.6",
     opts: {
       cacheDir: CACHE_DIR,
       remoteUrl:
-        "https://storage.googleapis.com/firebase-preview-drop/emulator/cloud-firestore-emulator-v1.11.5.jar",
-      expectedSize: 63287761,
-      expectedChecksum: "badd7007623ce51761d7a8ec2be42f8f",
+        "https://storage.googleapis.com/firebase-preview-drop/emulator/cloud-firestore-emulator-v1.11.6.jar",
+      expectedSize: 91311475,
```
Comment (yuchenshi) [inline src/emulator/downloadableEmulators.ts:49]: ~30MB size increase? I wonder what is going on here...

File: CHANGELOG.md (hunk 1 above)
Comment (malcolmdeck) [reply CHANGELOG.md]: Done.

File: src/emulator/downloadableEmulators.ts (hunk 2 above)
Comment (malcolmdeck) [reply src/emulator/downloadableEmulators.ts]: Resolved offline - command is missing "-c opt"; Yuchen fixed the build script :)

**Pull request merged**